
WARNING:tensorflow:From train.py:75: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.
WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.
WARNING:tensorflow:From C:\Users\aplus\macbf\drones\drones\core.py:278: The name tf.AUTO_REUSE is deprecated. Please use tf.compat.v1.AUTO_REUSE instead.
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E50D4EF128>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E50D4EF128>>: AttributeError: module 'gast' has no attribute 'Str'
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E531395F60>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E531395F60>>: AttributeError: module 'gast' has no attribute 'Str'
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E50D4EF438>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E50D4EF438>>: AttributeError: module 'gast' has no attribute 'Str'
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E5313B6080>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E5313B6080>>: AttributeError: module 'gast' has no attribute 'Str'
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E531564BE0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E531564BE0>>: AttributeError: module 'gast' has no attribute 'Str'
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E531537B38>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E531537B38>>: AttributeError: module 'gast' has no attribute 'Str'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x000001E531580C50>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x000001E531580C50>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x000001E531580A20>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x000001E531580A20>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x000001E531580A20>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x000001E531580A20>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x000001E531564BE0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x000001E531564BE0>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:From D:\Anaconda\envs\macbf\lib\site-packages\tensorflow\python\ops\array_ops.py:1354: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E53276CEF0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E53276CEF0>>: AttributeError: module 'gast' has no attribute 'Str'
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E53277FE80>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E53277FE80>>: AttributeError: module 'gast' has no attribute 'Str'
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E53277F588>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E53277F588>>: AttributeError: module 'gast' has no attribute 'Str'
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E5327BFC88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x000001E5327BFC88>>: AttributeError: module 'gast' has no attribute 'Str'
WARNING:tensorflow:From train.py:120: The name tf.trainable_variables is deprecated. Please use tf.compat.v1.trainable_variables instead.
WARNING:tensorflow:From train.py:28: The name tf.train.AdamOptimizer is deprecated. Please use tf.compat.v1.train.AdamOptimizer instead.
WARNING:tensorflow:From train.py:36: Variable.initialized_value (from tensorflow.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Use Variable.read_value. Variables in 2.X are initialized automatically both in eager and graph (inside tf.defun) contexts.
WARNING:tensorflow:From train.py:150: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.
WARNING:tensorflow:From train.py:151: The name tf.global_variables_initializer is deprecated. Please use tf.compat.v1.global_variables_initializer instead.
WARNING:tensorflow:From train.py:152: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.
Step: 0, Time: 1.7, Loss: [6.479e-03 9.197e-04 1.465e-02 1.231e-04 5.321e-03 2.656e-02 2.661e+00
 3.640e-02], Dist: 0.536, Safety Rate: 1.000
<class 'numpy.float32'>
Step: 1, Time: 1.1, Loss: [3.078e-02 1.005e-02 1.346e-01 5.154e-04 3.194e-02 2.466e-02 2.455e+00
 9.871e-02], Dist: 1.016, Safety Rate: 0.716
<class 'numpy.float32'>
D:\Anaconda\envs\macbf\lib\site-packages\numpy\core\fromnumeric.py:3373: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
D:\Anaconda\envs\macbf\lib\site-packages\numpy\core\_methods.py:170: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Step: 2, Time: 1.0, Loss: [2.616e-03 7.922e-04 1.843e-02 7.111e-05 1.492e-02 2.205e-02 2.711e+00
 4.163e-02], Dist: 0.493, Safety Rate: 0.975
<class 'numpy.float32'>
Step: 3, Time: 0.9, Loss: [0.000e+00 9.576e-04 0.000e+00 1.043e-04 0.000e+00 2.862e-02 2.674e+00
 5.306e-02], Dist: 0.801, Safety Rate: 1.000
<class 'numpy.float32'>
Step: 4, Time: 0.8, Loss: [7.688e-03 7.126e-04 2.915e-02 1.113e-04 2.008e-03 2.398e-02 2.681e+00
 5.723e-02], Dist: 0.902, Safety Rate: 0.954
<class 'numpy.float32'>
Step: 5, Time: 1.1, Loss: [0.000e+00 7.949e-04 0.000e+00 1.512e-04 7.330e-03 2.156e-02 2.650e+00
 6.114e-02], Dist: 0.437, Safety Rate: 1.000
<class 'numpy.float32'>
Step: 6, Time: 1.1, Loss: [9.294e-03 1.365e-03 1.038e-01 1.260e-04 1.822e-02 2.218e-02 2.695e+00
 3.429e-02], Dist: 0.371, Safety Rate: 0.822
<class 'numpy.float32'>
Step: 7, Time: 1.0, Loss: [0.000e+00 9.273e-04 0.000e+00 5.798e-05 0.000e+00 2.797e-02 2.670e+00
 3.458e-02], Dist: 0.590, Safety Rate: 1.000
<class 'numpy.float32'>
Step: 8, Time: 1.0, Loss: [0.000e+00 1.624e-03 0.000e+00 1.221e-04 0.000e+00 1.919e-02 2.673e+00
 3.874e-02], Dist: 0.927, Safety Rate: 1.000
<class 'numpy.float32'>
Step: 9, Time: 1.0, Loss: [2.453e-03 1.711e-03 1.181e-01 2.207e-04 1.016e-02 2.965e-02 2.474e+00
 9.386e-02], Dist: 0.515, Safety Rate: 0.804
<class 'numpy.float32'>
Step: 10, Time: 1.1, Loss: [0.000e+00 1.379e-03 0.000e+00 1.101e-04 8.074e-03 2.647e-02 2.612e+00
 4.685e-02], Dist: 0.488, Safety Rate: 1.000
